

 var project_info=[
    {
        "uuid":"ffc3e90f-7f89-4934-8e35-943bb64d1789",
        "authors": ["Anthony Hall", "Laura-Jayne Gardiner"],
        "projectName": "Integrating genomic resources to present full gene and promoter capture probe sets for bread wheat",
        "description": "Whole genome shotgun re-sequencing of wheat is expensive because of its large, repetitive genome. Moreover, sequence data can fail to map uniquely to the reference genome making it difficult to unambiguously assign variation. Re-sequencing using target capture enables sequencing of large numbers of individuals at high coverage to reliably identify variants associated with important agronomic traits. We present two gold standard capture probe sets for hexaploid bread wheat, a gene and a promoter capture, which are designed using recently developed genome sequence and annotation resources. The captures can be combined or used independently. The capture probe sets effectively enrich the high confidence genes and promoters that were identified in the genome alongside a large proportion of the low confidence genes and promoters. We use a capture design employing an 'island strategy' to enable analysis of the large gene/promoter space of wheat with only 2x160 Mb NimbelGen probe sets. Furthermore, these assays extend the regions of the wheat genome that are amenable to analyses beyond its exome, providing tools for detailed characterization of these regulatory regions in large populations. Here, we release the targeted sequence of the capture probe sets on the wheat RefSeqv1, the design space that was used to tile our capture probes across and finally the positions of the probes themselves across this design space for both the gene and promoter capture probe sets. This project was supported by the BBSRC via an ERA-CAPS grant BB/N005104/1, BB/N005155/1 and BBSRC Designing Future Wheat BB/P016855/1.",
        "license" : {"so:name": "toronto", "so:url": "https://www.nature.com/articles/461168a#Sec2" },
        "project_codes":["BB/N005104/1", "BB/N005155/1", "BB/P016855/1"],
        "url":"https://opendata.earlham.ac.uk/wheat/under_license/toronto/Gardiner_2018-07-04_Wheat-gene-promoter-capture/",
        "irods_path":"/grassrootsZone/public/under_license/toronto/Gardiner_2018-07-04_Wheat-gene-promoter-capture",
        "@type": "Grassroots:project",
        "type_description": "Dataset",
        "so:image": "https://grassroots.tools/grassroots/images/aiss/drawer"
    },
    {
        "uuid":"fd22c6a9-5692-4bbd-bb77-e04567fe3bc0",
        "authors": ["Ji Zhou", "Joshua Ball"],
        "projectName": "DFW RobigusxClaire UAV Image Data",
        "description": "Working with the Griffiths group at JIC we have collected UAV image data for the RobigusxClaire population across the 2019 season. Divided into early, mid, and late drilling dates. Raw images from UAV capture are pre-processed using Pix4DMapper software to create a large, high-detail, orthomosaic images providing a single image for analysis and extraction of wheat traits for QTL analysis. Single whole field images are also captured to account for light and colour variations, This project is supported by the BBSRC Designing Future Wheat grant sub-work package BBS/E/T/000PR9785",
        "license" : {"so:name": "toronto", "so:url": "https://www.nature.com/articles/461168a#Sec2" },
        "project_codes":["BBS/E/T/000PR9785"],
        "url":"https://opendata.earlham.ac.uk/wheat/under_license/toronto/Zhou_2019_RobxCla_UAV_Image_Data/",
        "irods_path":"/grassrootsZone/public/under_license/toronto/Zhou_2019_RobxCla_UAV_Image_Data",
        "@type": "Grassroots:project",
        "type_description": "Dataset",
        "so:image": "https://grassroots.tools/grassroots/images/aiss/drawer"
    },
    {
        "uuid":"4c53dd08-39d6-4d3a-bfc0-c6a5f32b609f",
        "authors": ["Azahara Carmen Martín", "AbdulKader Alabdullah", "Philippa Borrill", "Ricardo H. Ramírez-González", "Janet Higgins", "David Swarbreck", "Cristobal Uauy", "Peter Shaw", "Graham Moore"],
        "projectName":"Meiosis transcriptome and co-expression network in hexaploid wheat",
        "description": "Despite its large genome size, high DNA repetitive content and similarity between homoeologous chromosomes, hexaploid wheat completes meiosis in a shorter period than diploid species with a much smaller genome. Therefore, during wheat meiosis, mechanisms additional to the classical model based on DNA sequence homology, must facilitate more efficient homologous recognition. One such mechanism could involve exploitation of differences in chromosome structure between homologs and homoeologs at the onset of meiosis. In turn, these chromatin changes can be expected to be linked to transcriptional gene activity. In this study, we present an extensive analysis of a large RNA-seq data derived from 6 different genotypes: wheat, wheat-rye hybrids and newly synthesized octoploid triticale, both in the presence and absence of the Ph1 locus. Plant material was collected at early prophase, at the transition leptotene-zygotene, when the six genotypes exhibit different levels of synapsis and chromatin structure. Unexpectedly, our study reveals that neither synapsis, whole genome duplication nor the absence of the Ph1 locus are associated with major changes in gene expression levels during early meiotic prophase. Overall wheat transcription at this meiotic stage is therefore highly resilient to such alterations, even in the presence of major chromatin structural changes. Our knowledge of the genes involved in meiosis in many crop species such as wheat is largely based on studies on model species. Here we used the WGCNA package in R to build a meiotic gene co-expression network in wheat based on 130 wheat RNA-seq samples collected from a range of tissues including meiotic anthers. A set of 50,387 genes were expressed during meiosis and assigned to 66 modules according to their expression patterns. Three of the modules (modules 2, 28 and 41 containing 4940 genes, 544 genes and 313 genes, respectively) were significantly correlated with meiotic tissue samples but not with any other type of tissue. The three meiosis-related modules were highly enriched with GO terms related to cell cycle, DNA replication, chromatin modifications and other processes occurring during meiosis. Wheat orthologs of meiosis genes were found in modules 2, 28 and 41. Module 2, in particular, was significantly enriched possessing 166 meiosis orthologs. The combination of co-expression network analysis in tandem with orthologue information will contribute to enhance wheat meiotic studies, which will lead to better understanding of the regulation of meiosis in wheat (and other polyploid plants) and subsequently improve wheat production. Project Code: BB/J007188/1.",
        "license" : {"so:name": "toronto", "so:url": "https://www.nature.com/articles/461168a#Sec2" },
        "project_codes":["BB/J007188/1"],
        "url":"https://opendata.earlham.ac.uk/wheat/under_license/toronto/Martin_etal_2018_Alabdullah_etal_2019_wheat_meiosis_transcriptome_and_co-expression_network/",
        "irods_path":"/grassrootsZone/public/under_license/toronto/Martin_etal_2018_Alabdullah_etal_2019_wheat_meiosis_transcriptome_and_co-expression_network",
        "@type": "Grassroots:project",
        "type_description": "Dataset",
        "so:image": "https://grassroots.tools/grassroots/images/aiss/drawer"
    },
    {
        "uuid":"844c0d71-5288-4f85-ae66-cc7d66596ab1",
        "authors": ["Brande Wulff", "Burkhard Steuernagel"],
        "projectName": "Aegilops tauschii diversity panel",
        "description": "This is the raw data from whole genome shotgun sequencing of 150 accessions of Aegilops tauschii, a wild relative of wheat. This data was generated by the open wild wheat consortium. Financial support for this project includes (but is not excluded to) the BBSRC (Designing Future Wheat BB/P016855/1), KWS, Syngenta, Bayer and Limagrain. In kind support was received from CLC Bio, Novogene and 2Blades.",
        "license" : {"so:name": "toronto", "so:url": "https://www.nature.com/articles/461168a#Sec2" },
        "project_codes":["BB/P016855/1"],
        "url":"https://opendata.earlham.ac.uk/wheat/under_license/toronto/Wulff_2018-01-31_OWWC/",
        "irods_path":"/grassrootsZone/public/under_license/toronto/Wulff_2018-01-31_OWWC",
        "@type": "Grassroots:project",
        "type_description": "Dataset",
        "so:image": "https://grassroots.tools/grassroots/images/aiss/drawer"
    },
    {
        "uuid":"1e949203-127e-4d5d-8738-fa1e98fb4b82",
        "authors": ["Ricardo Ramirez-Gonzalez", "Philippa Borrill", "Cristobal Uauy"],
        "projectName": "The transcriptional landscape of hexaploid wheat across tissues, cultivars, and stress conditions",
        "description": "The coordinated expression of highly related homoeologous genes in polyploid species underlies the phenotypes of many of the world?s major crops. However, the balance of homoeolog expression across diverse tissues, stress conditions, and cultivars remains poorly understood. Here we combine extensive gene expression datasets with the fully annotated genome sequence to produce a comprehensive, genome-wide analysis of homoeolog expression patterns in hexaploid bread wheat. Bias in homoeolog expression varied between tissues, with ~30% of wheat homoeologs showing unbalanced expression. We found expression asymmetries along wheat chromosomes, with genes showing the largest inter-tissue, inter-cultivar, and coding sequence variation most often located in the high-recombination distal ends of chromosomes. These transcriptionally dynamic genes potentially represent the first steps towards neo/sub- functionalization of wheat homoeologs. Co-expression networks revealed extensive coordination of homoeologs throughout development and, alongside a detailed expression atlas, provide a framework to target candidate genes underpinning agronomic traits in polyploid wheat. Project Code: BB/P016855/1",
        "license" : {"so:name": "toronto", "so:url": "https://www.nature.com/articles/461168a#Sec2" },
        "project_codes":["BB/P016855/1"],
        "url":"https://opendata.earlham.ac.uk/wheat/under_license/toronto/Ramirez-Gonzalez_etal_2018-06025-Transcriptome-Landscape/",
        "irods_path":"/grassrootsZone/public/under_license/toronto/Ramirez-Gonzalez_etal_2018-06025-Transcriptome-Landscape",
        "@type": "Grassroots:project",
        "type_description": "Dataset",
        "so:image": "https://grassroots.tools/grassroots/images/aiss/drawer"
    },{
        "uuid":"5dd4fd26-56f7-4ae6-a046-70969f084ce8",
        "authors": ["Anthony Hall", "Laura-Jayne Gardiner"],
        "projectName": "Using 12Mb capture to profile the Watkins bread wheat landrace diversity collection",
        "description": "Wheat has been domesticated into a large number of agricultural environments and has a remarkable ability to adapt to diverse environments. To understand this process, we survey genotype and DNA methylation across the Watkins bread wheat landrace collection, representing global genetic diversity. For each accession, we use gene based sequence capture (12Mb) to focus on the functionally relevant portion of the genome, followed by paired end sequencing on the Hiseq4000. All Watkins accessions therefore have genotypic data available plus sequence data generated after bisulfite treatment to allow methylation calls. This project was supported by the BBSRC via an ERA-CAPS grant BB/N005104/1, BB/N005155/1 and BBSRC Designing Future Wheat BB/P016855/1.",
        "license" : {"so:name": "toronto", "so:url": "https://www.nature.com/articles/461168a#Sec2" },
        "project_codes":["BB/N005104/1", "BB/N005155/1", "BB/P016855/1"],
        "url":"https://opendata.earlham.ac.uk/wheat/under_license/toronto/Gardiner_2018-01-29_Watkins-diversity-12Mb/",
        "irods_path":"/grassrootsZone/public/under_license/toronto/Gardiner_2018-01-29_Watkins-diversity-12Mb",
        "@type": "Grassroots:project",
         "type_description": "Dataset",
         "so:image": "https://grassroots.tools/grassroots/images/aiss/drawer"
    },{
         "uuid":"c0d214a8-f09e-4f15-bcd2-f33e023a159b",
         "authors": ["Luzie U. Wingen", "Simon Orford", "Michelle Leverington-Waite", "Sarah Collier", "Rajani Awal", "Simon Griffiths"],
         "projectName": "DFW-WP3: Germplasm Development for Trait Dissection - Initial QTL Validation",
         "description": "Phenotype data for QTL beneficial allele validation using Near Isogenic Lines (NILs) from growing season Oct 2017 - Aug 2018 at Church Farm, John Innes Centre, Norwich, UK, for the traits: heading time, plant height and  grain yield. Phenotype data was collected for 189 NILs of the DFW-TK-Landrace-NIL-Set2, which were developed from six landrace parents (Watkins 1190034, Watkins 1190141, Watkins 1190292, Watkins 1190352, Watkins 1190468, Watkins 1190729) and cultivar 'Paragon'. The introgressed 37 QTL, present show association with one of the following traits: above ground dry mass, anthesis date, grain fill, grain yield, height, lodging, maturity mildew score, NDRE, NDVI, and straw yield. Furthermore, phenotype data for two DFW-TK-Mutation-NILs,  from a mutation screen for gw2-A, and for the control cultivars 'Paragon' and 'Soisson' was also collected. The DFW Wheat Toolkit (TK) contains sets of pre-breeding germplasm lines, developed by Work Package 3 of the Designing Future Wheat (DFW) Institute Strategic Programme (https://designingfuturewheat.org.uk/work-package-3/) and includes NILs or equivalent material from different wheat diversity sources, e.g. landraces, mutation screening, ancient wheat relatives and new synthetic wheats. DFW TK lines  are assessed for traits of interest within the DFW programme. Genotype data, for SNPs on the 35k Axiom Wheat Genotyping Breeders' Array, is available from CerealsDB.",
         "license" : {"so:name": "toronto", "so:url": "https://www.nature.com/articles/461168a#Sec2" },
         "project_codes":["BB/P016855/1"],
         "url":"https://opendata.earlham.ac.uk/wheat/under_license/toronto/Griffiths_2018-10-22_DFW-TK-NIL-Set2-2018-phenotypes/",
         "irods_path":"/grassrootsZone/public/under_license/toronto/Griffiths_2018-10-22_DFW-TK-NIL-Set2-2018-phenotypes",
         "@type": "Grassroots:project",
         "type_description": "Dataset",
         "so:image": "https://grassroots.tools/grassroots/images/aiss/drawer"
     },{
         "uuid":"1f0ed4e0-be12-49b1-a65e-cab36a449f95",
         "authors": ["Malcolm Hawkesford", "Andrew Riche"],
         "projectName": "DFW Academic Toolkit Harvest 2020",
         "description": "Within Workpackage 3 of Designing Future Wheat, there is an Academic Toolkit field experiment each year, which provides information for selecting a subset of lines to constitute the Breeders toolkit, which is grown the following year by the UK wheat breeding community. An image was taken of each plot, post anthesis, to show what the lines look like, and give an idea of purity.",
         "license" : {"so:name": "toronto", "so:url": "https://www.nature.com/articles/461168a#Sec2" },
         "project_codes":["BB/P016855/1"],
         "url":"https://opendata.earlham.ac.uk/wheat/under_license/toronto/Riche_2020-07-13_DFW_Academic_Toolkit_Harvest_2020/",
         "irods_path":"/grassrootsZone/public/under_license/toronto/Riche_2020-07-13_DFW_Academic_Toolkit_Harvest_2020",
         "@type": "Grassroots:project",
         "type_description": "Dataset",
         "so:image": "https://grassroots.tools/grassroots/images/aiss/drawer"
     }
]